exercicios/fizz-buzz.csv

```diff
@@ -0,0 +1,4 @@
+"Imprima Fizz se um numero for multiplo de 3 e Buzz se for multiplo de 5. Se for multiplo de 3 e 5 imprima FizzBuzz. Se não for multiplo de nenhum dos dois, nao imprima nada.", "3", "Fizz"
```

Comment:
FizzBuzz = 1,2,"Fizz",4,"Buzz"
Qaudno nao for multiplo de 3 ou 5, imprime o proprio numero.